toolbox/org.lamport.tla.toolbox/src/org/lamport/tla/toolbox/spec/parser/ModuleParserLauncher.java

```diff
@@ -536,7 +537,7 @@ else if (message.indexOf("Lexical {error: EOF reached, possibly open comment sta
             if (result.getSemanticErrors() != null)
             {
 
-                String[][] errors = { result.getSemanticErrors().getErrors(), result.getSemanticErrors().getWarnings() };
+                String[][] errors = { result.getSemanticErrors().getErrors(), result.getSemanticErrors().getMessagesOfAtLeastLevel(ErrorLevel.LINT).stream().map(ErrorDetails::toString).toArray(String[]::new) };

```

Comment:
This is a bit unwieldy but really the entire loop below this should be refactored to use ErrorDetails. It parses the string error message to extract the location.